libcontainer/container_linux.go

```diff
@@ -778,6 +790,18 @@ func (c *linuxContainer) Restore(process *Process, criuOpts *CriuOpts) error {
 		}
 	}
 
+	for _, path := range c.config.MaskPaths {
+		m := &configs.Mount{Destination: path, Source: "/dev/null"}
+		c.addCriuRestoreMount(req, m)
+	}
+
+	if c.config.Namespaces.Contains(configs.NEWUSER) {
```

Comment:
I don't really know enough about this to know the answer, but is it definitely the case that we only want to add mounts for c.config.Devices if there's a `newuser` namespace?